src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -3541,32 +3403,15 @@ public List<EnhancedKeyUsageRepresentation> EnhancedKeyUsageList
         /// </summary>
         public EnhancedKeyUsageProperty(X509Certificate2 cert)
         {
-            if (DownLevelHelper.NativeFilteringSupported())
+            foreach (X509Extension extension in cert.Extensions)
             {
-                Collection<string> ekuCollection = System.Management.Automation.Internal.SecuritySupport.GetCertEKU(cert);
-
-                foreach (string oidString in ekuCollection)
+                if (extension.Oid.FriendlyName == "Enhanced Key Usage")
```

Comment:
Except that is the OID for a specific EKU, not specifying an EKU.  The OID should be 2.5.29.37